python_modules/libraries/dagster-dg/dagster_dg/cli/global_options.py

```diff
@@ -36,6 +36,12 @@
             default=DgConfig.builtin_component_lib,
             help="Specify a builitin component library to use.",
         ),
+        click.Option(
+            ["--disable-uv"],
```

Comment:
Alternatives could be `--disable-uv-cache`, `--disable-uv-venv`?

Reply:
New name is `--manage-environment`/`--no-manage-environment`